config/kubernetes/org.yaml

```diff
@@ -1706,30 +1706,32 @@ teams:
   autoscaler-admins:
```

Comment:
Can you move these teams to `config/kubernetes/sig-autoscaling/teams.yaml` so that sig autoscaling leads can approve any future changes to these teams directly?

Reply:
Sure, now done as a separate commit.